bot/controllers/verify.js

```diff
@@ -53,11 +62,20 @@ module.exports = () => {
               if (data === null) {
                 // no existing record found
                 models.Member.create({
-                  discorduser: m.author.username,
+                  discorduser: m.author.id,
                   email,
                   uuid: uuidv4(),
                   verified: 1,
-                }).then(util.log).catch(util.error);
+                });
+                // maping guild roles to find the crew role id
+                const guild = message.guild.roles.map(channel => channel);
+                Object.keys(guild).forEach((el) => {
+                  if (guild[el].name === 'crew') {
+                    m.member.addRole(guild[el].id);
+                  } else {
+                    util.log('the moredator has to create the crew role');
```

Comment:
Typo: moredator -> moderator; Also, no `output` and `level` params passed to `util.log()`. Recommend `util.log(<message>, el, 2);`